source/sdk/io/File.ooc

```diff
@@ -18,604 +18,603 @@ import text/StringTokenizer
  */
 File: abstract class {
 
-    /** The path we're representing */
-    path: String { get set }
-
-    name: String { get { getName() } }
-    parent: This { get { getParent() } }
-    
-    extension: String { get { getExtension() } }
-    nameWithoutExtension: String { get { getNameWithoutExtension() } }
-    pathWithoutExtension: String { get { getPathWithoutExtension() } }
-    rectifySeparator: func {
-		if (this dir?() && !this path endsWith?(File separator)) {
+	/** The path we're representing */
+	path: String { get set }
+
+	name: String { get { getName() } }
+	parent: This { get { getParent() } }
+
+	extension: String { get { getExtension() } }
+	nameWithoutExtension: String { get { getNameWithoutExtension() } }
+	pathWithoutExtension: String { get { getPathWithoutExtension() } }
+	rectifySeparator: func {
+		if (this dir?() && !this path endsWith?(This separator)) {
 			oldPath := this path
-			this path = oldPath + File separator
+			this path = oldPath + This separator
 			oldPath free()
```

Comment:
`this path = this path >> This separator` should work as a replacement to the 3 lines here.